Utilities/BEType.h

```diff
@@ -403,7 +402,7 @@ inline v128 operator^(const v128& left, const v128& right)
 
 inline v128 operator~(const v128& other)
 {
-	return v128::from64(~other._u64[0], ~other._u64[1]);
+	return other ^ v128::eq32(other, other); // XOR with ones
```

Comment:
Normally most compilers would emit "all ones" constant like this anyway, but by explicitly telling it how to do it actually reduces the possibility of optimizations.